src/containers/App/App.test.js

```diff
@@ -23,39 +29,113 @@ describe('App', () => {
     let wrapper;
     let mockData;
     let mockSetMovies;
+    let mockUser;
+    let mockError;
+    let mockSetUser;
+    let mockLoading;
 
     beforeEach(() => {
+      mockLoading = jest.fn();
+      mockSetUser = jest.fn();
       mockSetMovies = jest.fn();
       mockData = { results: [{ name: 'AquaMan', id: 1 }, { name: 'Beauty and the Beast', id: 2 }] };
-      API.fetchData = jest.fn().mockImplementation(() => {
+      mockUser = { name: 'Tommy', email: 'a@a', id: 1, password: 'password', favorites: [] };
+      mockError = jest.fn();
+      API.getData = jest.fn().mockImplementation(() => {
         return mockData
       });
       wrapper = shallow(
       <App 
+        user={mockUser}
+        movies={mockData}
         setMovies={mockSetMovies}
+        errorToDisplay={mockError}
+        setCurrentUser={mockSetUser}
+        isLoading={mockLoading}
       />);
     });
 
-    it('should call fetchData with the correct parameters', async () => {
+    it('should call getData with the correct parameters', async () => {
       const expected = `https://api.themoviedb.org/3/movie/popular?page=1&api_key=${apiKey}&language=en-US`;
       await wrapper.instance().componentDidMount();
-      expect(API.fetchData).toHaveBeenCalledWith(expected);
+      expect(API.getData).toHaveBeenCalledWith(expected);
```

Comment:
I like the change to get from fetch, like you had said it aligns with the get/post actions